frontstage/views/secure_messaging.py

```diff
@@ -160,32 +161,20 @@ def get_survey_id(party_id):
     return survey_name
 
 
-def get_messages(label, logger):
-    headers = {"Authorization": request.cookies['authorization']}
+def get_messages_list(label, logger):
```

Comment:
I know you didn't add it but why are we passing in the logger?

Reply:
Rather than defining `party_id`, `message_id` etc in every relevant log line they get bound to a logger with `logger.bind`, e.g. on line 26. From that point onwards everywhere that logger is called with include `party_id`. 
Once I bind anything to a logger I pass it through the other functions so it retains these details without having to explicitly write them out on every log line

Reply:
I think you get that behaviour without assigning to logger and passing it through

Reply:
I'll test it out now

Reply:
Just had a play around with it and I couldn't recreate the behaviour without doing it the way it's currently being implemented. If we wanting to change this should we raise it as another ticket rather than doing it as part of this request?

Reply:
Cool it's strange I was sure it did when I did this on EQ (https://github.com/ONSdigital/eq-survey-runner/blob/master/app/views/questionnaire.py#L49) but if it's not a quick one lets investigate on another ticket

Reply:
That is a bit strange, we do define the logger slightly differently so that may be it. 
`logger = get_logger()` vs `logger = wrap_logger(logging.getLogger(__name__))`
Either way there's quite a few places we'd have to change it so probably should do on another card.